src/complex.rs

```diff
@@ -59,6 +59,14 @@ impl<T: Clone + Num> Complex<T> {
     }
 }
 
+impl<T: Clone + Num> Complex<T> {
+    /// Returns `i`
+    #[inline]
+    pub fn i() -> Complex<T> {
+        Self::new(T::zero(), T::one())
+    }
+}
```

Comment:
Accidentally as I firstly wrote that for types that are `Clone + Zero + One`, not `Clone + Num`.